src/overlays/actors/ovl_En_Go/z_en_go.c

```diff
@@ -1968,60 +2539,65 @@ void func_80A15B80(EnGo* this, PlayState* play) {
     Matrix_RotateXS(this->actor.shape.rot.x, MTXMODE_APPLY);
 
     gSPMatrix(POLY_OPA_DISP++, Matrix_NewMtx(play->state.gfxCtx), G_MTX_NOPUSH | G_MTX_LOAD | G_MTX_MODELVIEW);
-    gSPDisplayList(POLY_OPA_DISP++, (this->unk_390 & 0x100) ? gGoronSnowballDL : gGoronRolledUpDL);
+    gSPDisplayList(POLY_OPA_DISP++, (this->actionFlags & ENGO_FLAG_SNOWBALLED) ? gGoronSnowballDL : gGoronRolledUpDL);
 
     CLOSE_DISPS(play->state.gfxCtx);
 }
 
+/**
+ * OverrideLimbDrawOpa function for the Gorons.
+ */
```

Comment:
Ditto, rule of thumb is that if it doesn't add anything to the function name then it's not really worth including.